api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/AddressTest.java

```diff
@@ -87,6 +87,16 @@ void shouldFailToValidateAddressWhenTownOrCityIsTooLong() {
         assertThat(violations).hasSingleConstraintViolation("size must be between 1 and 500", "townOrCity");
     }
 
+    @Test
+    void shouldFailToValidateAddressWhenCountyIsTooLong() {
```

Comment:
Can we have a test that says there are no validations when the county is missing, or is that covered by the default success test here? Either way we should have a test with and without the county.

Reply:
Done.